src/main/java/io/openliberty/tools/intellij/lsp4mp4ij/psi/core/command/MicroprofileOpenURIAction.java

```diff
@@ -3,8 +3,8 @@
 import com.google.gson.JsonPrimitive;
 import com.intellij.ide.BrowserUtil;
 import com.intellij.openapi.actionSystem.AnActionEvent;
-import org.microshed.lsp4ij.commands.LSPCommand;
-import org.microshed.lsp4ij.commands.LSPCommandAction;
+import com.redhat.devtools.lsp4ij.commands.LSPCommand;
+import  com.redhat.devtools.lsp4ij.commands.LSPCommandAction;
```

Comment:
If someone has to make a pull request to fix anything they should tidy this as well.